samples/bluetooth/fast_pair/input_device/prj.conf

```diff
@@ -14,10 +14,6 @@ CONFIG_DK_LIBRARY=y
 # Enable settings to store data in non-volatile memory
 CONFIG_FLASH=y
 CONFIG_FLASH_MAP=y
-CONFIG_NVS=y
-CONFIG_SETTINGS_NVS=y
-CONFIG_SETTINGS=y
-CONFIG_BT_SETTINGS=y
```

Comment:
Could you please restore the `CONFIG_BT_SETTINGS=y` line?
It seems to have been removed by mistake - it is needed for this sample.